js/runner.go

```diff
@@ -140,7 +173,15 @@ func (r *Runner) newVU(samplesOut chan<- stats.SampleContainer) (*VU, error) {
 			nameToCert[name] = &certs[i]
 		}
 	}
-
+	if r.Bundle.Options.Nic.ValueOrZero() != "" {
```

Comment:
I'd suggest that checking if `r.Bundle.Options.Nic.Valid` is `true` would be a better approach here. The whole idea behind the `null`-able types we use in the k6 config is that you can differentiated between "value is not set" and "value is set and is actually the empty value for the type" . The [`null.String`](https://godoc.org/gopkg.in/guregu/null.v3#String) type is actually a simple wrapper around the [`NullString`](https://golang.org/pkg/database/sql/#NullString) in the go `sql` package.

Right now, with `ValueOrZero`, even if you didn't suppress the errors in `rr_getAddr()`, if users ran with `k6 run --nic ""`, no error would be shown...

Reply:
Fixed